rules/qc.smk

```diff
@@ -0,0 +1,225 @@
+## RSEQC
+rule rseqc_gtf2bed12:
+    input:
+        config["ref"]["annotation"]
+    output:
+        "star/rseqc/annotation.bed"
+    log:
+        "logs/gtf2bed12.stderr"
+    params:
+        script = "../scripts/gtf2bed.pl"
+    shell:
+        """
+        perl {params.script} {input} > {output} 2> {log}
```

Comment:
I think we should use bedops here instead of a custom included script: https://bedops.readthedocs.io/en/latest/content/reference/file-management/conversion/gtf2bed.html

Reply:
ok, but it will be `gff2bed` as it seems the annotation in .test/data is in gff3 format... `gtf2bed` throws errors.

Reply:
I also get errors running it with gff2bed.. I will have a look why this is...

Reply:
seems to be gtf afterall however, still not sure why gtf2bed throws error.

Reply:
ok rseqc needs bed12. that was also the reason I originally did use the script, as I could not find any "bioconda" tool that could do that... ideas welcome.

Reply:
BED format has different versions as well? Can you point me to the specs?

Reply:
[https://genome.ucsc.edu/FAQ/FAQformat.html#format1](https://genome.ucsc.edu/FAQ/FAQformat.html#format1).

Important is the `blockStarts` I believe as these indicate for each tx where the exons starts and this info is used by `rseqc` (badly documented though).